release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -2456,6 +2456,8 @@ openshift.io/scc: hostmount-anyuid
 
 * There is a known issue on Nutanix where persistent volumes (PVs) created by a cluster are not cleaned up by the  `destroy cluster` command. As a workaround for this issue, you need to clean up the PVs manually. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2108700[*BZ#2108700*])
 
+* There is a known issue with Nutanix installation where the installation fails if you use 4096-bit certificates with Prism Central 2022.x. Instead, use 2048-bit certificates. (link:https://access.redhat.com/solutions/6976743[Red Hat Customer Portal solution])
```

Comment:
Small note based on our slack convo

```suggestion
* There is a known issue with Nutanix installation where the installation fails if you use 4096-bit certificates with Prism Central 2022.x. Instead, use 2048-bit certificates. (link:https://access.redhat.com/solutions/6976743[*KCS*])
```

Reply:
@pneedle-rh Thanks for your peer review. I put "KCS" after asking Olivia, and she brought it up with the other Release Notes folks since there isn't another case like it. They recommended to just use KCS.